src/sticker/artist.py

```diff
@@ -57,6 +59,7 @@ def draw_description_sticker(self, description: str) -> tuple[str, bytes]:
             self.OUTER_GROUP_STICKER_COLOR
         )
         image = self.__add_text_on_sticker(image, description, 0)
+        image = remove(image)
```

Comment:
Wrong method, this should be added to `draw_sticker_from_prompt` method from below:
> async def draw_sticker_from_prompt(self, prompt: str) -> tuple[str, bytes]:
        """Draws a achievement sticker from prompt"""
        ...

Also as discussed offline, probably before we've implemented any kind of sticker deletion method, would be nice to have some fallback from incorrect background removal, for now it would be number of non-transparent clusters = 1 check with a fallback to original sticker generation algorithm (without applied background removal algorithm)

Reply:
Sure!
Already done